src/OpenLoco/Windows/BuildVehicle.cpp

```diff
@@ -1079,12 +1120,27 @@ namespace OpenLoco::Ui::Windows::BuildVehicle
                             continue;
                         }
 
+                        auto displayLockedVehiclesScroll = displayLockedVehicles && vehicleIsLocked(CompanyManager::getControllingId(), vehicleType);
                         auto colouredString = StringIds::black_stringid;
                         if (window.row_hover == vehicleType)
                         {
-                            Gfx::fillRect(context, 0, y, window.width, y + window.row_height - 1, 0x2000030);
+                            if (displayLockedVehiclesScroll)
+                            {
+                                Gfx::fillRect(context, 0, y, window.width, y + window.row_height - 1, 0x0100003D);
```

Comment:
Fwiw, the upper set of these are flags we haven't yet named in OpenLoco. For reference, see e.g.:

https://github.com/OpenRCT2/OpenRCT2/blob/d87089259e16a7d2651925ff67a78dad1b19586e/src/openrct2/drawing/Drawing.h#L188-L196

Reply:
this might help naming some?
https://docs.google.com/spreadsheets/d/15kOLly0F-Qkt7gaf2TEXx6GFUg0PolZKMeDsmcUgYxA/edit#gid=1418230292

Reply:
oh sorry my bad, I didn't realise you linked openrct2. I can name them in openloco in the next PR?